src/cpp/fastdds/xtypes/type_representation/dds_xtypes_typeobjectCdrAux.ipp

```diff
@@ -154,7 +154,7 @@ eProsima_user_DllExport void serialize(
     eprosima::fastcdr::Cdr::state current_state(scdr);
     scdr.begin_serialize_type(current_state,
             eprosima::fastcdr::CdrVersion::XCDRv2 == scdr.get_cdr_version() ?
-            eprosima::fastcdr::EncodingAlgorithmFlag::DELIMIT_CDR2 :
+            eprosima::fastcdr::EncodingAlgorithmFlag::PLAIN_CDR2 :
```

Comment:
why this is final now? I don't see changes in the IDL file.

Reply:
That structure is annotated in the IDL file as final. Nevertheless, that specific structure is used for Strongly Connected Components which are not yet supported.